plugin/ScriptLoader/ScriptLoaderGenericConfigurationDefault.php

```diff
@@ -67,12 +67,7 @@ public function getScriptLoaderGenericConfiguration(): array
 			(new IncludeFileModel($script, 'onoffice-form-preview', plugins_url('/js/onoffice-form-preview.js', $pluginPath)))
 				->setLoadInFooter(true),
 
-			new IncludeFileModel($style, 'onoffice-default', plugins_url('/css/onoffice-default.css', $pluginPath)),
```

Comment:
Deleting the old includes would make life really hard during development. What about this:
If the concatenated CSS file exists, load that, if it doesn't, load all the others.

Reply:
Yes, good idea!